disco/extensions/jade_extensions.json

```diff
@@ -11,14 +11,14 @@
         "cli_module": "disco.extensions.pydss_simulation.cli"
     },
     {
-        "name": "automated_upgrade_simulation",
-        "description": "Runs automated upgrade simulation using PyDSS on DISCO auto-generated configuration.",
-        "job_execution_module": "disco.extensions.automated_upgrade_simulation.automated_upgrade_simulation",
-        "job_execution_class": "AutomatedUpgradeSimulation",
-        "job_configuration_module": "disco.extensions.automated_upgrade_simulation.automated_upgrade_configuration",
-        "job_configuration_class": "AutomatedUpgradeConfiguration",
-        "job_parameters_module": "disco.distribution.deployment_parameters",
-        "job_parameters_class": "DeploymentParameters",
-        "cli_module": "disco.extensions.automated_upgrade_simulation.cli"
+        "name": "upgrade_simulation",
```

Comment:
Reminder to send a note to all users to tell them to delete ~/.jade-registry.json. Otherwise, this change will cause a failure. Or make a change to setup.py to remove the old entry.

Reply:
I updated the `setup.py` to remove the old registry file during the installation.